distr_tracing/distr_tracing_rn/distr-tracing-rn-2-9.adoc

```diff
@@ -0,0 +1,196 @@
+
+* Currently, when used with the {TempoOperator}, the Jaeger UI only displays services that have sent traces in the last 15 minutes. For services that did not send traces in the last 15 minutes, traces are still stored but not displayed in the Jaeger UI. (link:https://issues.redhat.com/browse/TRACING-3139[TRACING-3139])
+
+* Currently, the {TempoShortName} fails on the IBM Z (`s390x`) architecture. (link:https://issues.redhat.com/browse/TRACING-3545[TRACING-3545])
+
+* Currently, the Tempo query frontend service must not use internal mTLS when Gateway is not deployed. This issue does not affect the Jaeger Query API. The workaround is to disable mTLS. (link:https://issues.redhat.com/browse/TRACING-3510[TRACING-3510])
++
+.Workaround
+:!example-caption:
+====
+Disable mTLS as follows:
+
+. Open the {TempoOperator} ConfigMap for editing.
++
+[source,console]
+----
+$ oc edit configmap tempo-operator-manager-config -n openshift-tempo-operator <1>
+----
+<1> The project where the {TempoOperator} is installed.
+
+. Disable the mTLS in the Operator configuration.
++
+[source,yaml]
+----
+data:
+  controller_manager_config.yaml: |
+    featureGates:
+      httpEncryption: false
+      grpcEncryption: false
+      builtInCertManagement:
+        enabled: false
+----
+
+. Restart the {TempoOperator} Pod.
++
+[source,console]
+----
+$ oc rollout restart deployment.apps/tempo-operator-controller -n openshift-tempo-operator
+----
+====
+
+* Missing images for running the {TempoOperator} in restricted environments. The {TempoName} CSV is missing references to the operand images. (link:https://issues.redhat.com/browse/TRACING-3173[TRACING-3523])
++
+.Workaround
+:!example-caption:
+====
+
+Add the {TempoOperator} related images in the mirroring tool to mirror the images to the registry:
+
+[source,yaml]
+----
+kind: ImageSetConfiguration
+apiVersion: mirror.openshift.io/v1alpha2
+archiveSize: 20
+storageConfig:
+  local:
+    path: /home/user/images
+mirror:
+  operators:
+  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.13
+    packages:
+    - name: tempo-product
+      channels:
+      - name: stable
+  additionalImages:
+  - name: registry.redhat.io/rhosdt/tempo-rhel8@sha256:e4295f837066efb05bcc5897f31eb2bdbd81684a8c59d6f9498dd3590c62c12a
+  - name: registry.redhat.io/rhosdt/tempo-gateway-rhel8@sha256:b62f5cedfeb5907b638f14ca6aaeea50f41642980a8a6f87b7061e88d90fac23
+  - name: registry.redhat.io/rhosdt/tempo-gateway-opa-rhel8@sha256:8cd134deca47d6817b26566e272e6c3f75367653d589f5c90855c59b2fab01e9
+  - name: registry.redhat.io/rhosdt/tempo-query-rhel8@sha256:0da43034f440b8258a48a0697ba643b5643d48b615cdb882ac7f4f1f80aad08e
+----
+
+====
+
+== {OTELName}
+
+:FeatureName: The {OTELName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+=== New features and enhancements
+
+This release introduces the following noteworthy enhancements for the {OTELShortName}:
```

Comment:
```suggestion
This release introduces the following enhancements for the {OTELShortName}:
```

Reply:
Thank you. Already fixed in response to another comment.